scripts/adjust.js

```diff
@@ -42,7 +42,25 @@ const addSlateCoverImage = db.schema.table("slates", function (table) {
   table.jsonb("coverImage").nullable();
 });
 
-Promise.all([addSlateCoverImage]);
+const addSettingsColumnToUsersTable = db.schema
```

Comment:
What was the reason for adding a settings column? I removed all of these things earlier or moved them into another column already

Reply:
I added it back so I can store the filter sidebar state, I feel like storing it as a column will clutter the users' table. There is another option, we can create a settings table. I can also add to it the state of the sidebar sections if each section is collapsed or not